chia/wallet/trading/trade_store.py

```diff
@@ -13,6 +14,35 @@
 from chia.wallet.trading.trade_status import TradeStatus
 
 
+async def migrate_coin_of_interest(log: logging.Logger, db: aiosqlite.Connection) -> None:
```

Comment:
it seems it would be important for the creation of the table and the migration all be part of the same transaction. Is that the case?
If migration fails half-way through, will the table still have been created?